IssueComment
  { issueCommentUpdatedAt = 2014 (-06) (-16) 23 : 45 : 54 UTC
  , issueCommentUser =
      SimpleUser
        { simpleUserId = Id 2058304
        , simpleUserLogin = N "grandquista"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/2058304?v=3"
        , simpleUserUrl = "https://api.github.com/users/grandquista"
        , simpleUserType = OwnerUser
        }
  , issueCommentUrl =
      "https://api.github.com/repos/rethinkdb/docs/issues/comments/46251973"
  , issueCommentHtmlUrl =
      "https://github.com/rethinkdb/docs/pull/361#issuecomment-46251973"
  , issueCommentCreatedAt = 2014 (-06) (-16) 23 : 45 : 54 UTC
  , issueCommentBody =
      "It should also be noted that `docs.py` built on Python 2 will not work with Python 3.0 - 3.2."
  , issueCommentId = 46251973
  }